content/en/llm_observability/api.md

```diff
@@ -259,14 +257,16 @@ Evaluations require a `span_id` and `trace_id`.
         {
           "span_id": "61399242116139924211",
           "trace_id": "13932955089405749200",
-          "timestamp": 1609459200,
+          "ml_app": "weather-bot",
+          "timestamp_ms": 1609459200,
           "metric_type": "categorical",
           "label": "Sentiment",
           "categorical_value": "Positive"
         },
         {
           "span_id": "20245611112024561111",
           "trace_id": "13932955089405749200",
+          "ml_app": "weather-bot",
```

Comment:
does this need a timestamp_ms as well?

Reply:
or are we trying to show that timestamp is optional? i feel like we should push / encourage submission of timestamp regardless

Reply:
nah, it was a miss. timestamp is required, ty